pages/history.js

```diff
@@ -1,15 +1,21 @@
+import { s3 } from 'common/constants/urls';
+import Content from 'components/Content/Content';
 import Head from 'components/head';
 import HeroBanner from 'components/HeroBanner/HeroBanner';
-import Content from 'components/Content/Content';
 import Timeline from 'components/Timeline/Timeline';
-import { s3 } from 'common/constants/urls';
+import TimelineNav from 'components/Timeline/TimelineNav/TimelineNav';
 import styles from './styles/history.css';
 
 export default function() {
+  const pageContent = (
+    <section>
+      <TimelineNav />
+      <Timeline />
+    </section>
+  );
```

Comment:
I'd love if you inlined this as a Fragment on line 33

```jsx
<Content
  theme="white" 
  columns={[
    <>
      <TimelineNav />
      <Timeline />
    </>
  ]}
/>
```

Reply:
Unfortunately using a fragment seems to break the column width for the ```<Content />``` column. I could move the ```<TimelineNav />``` into the ```<TimeLine />``` component if needed. Let me know what your preference is 👍